docs/entities/servicebinding.md

```diff
@@ -17,22 +17,25 @@ The status of a Service Binding is also defined in our [ServiceBinding CRD](../.
 The state of the service binding can be `Malformed` or `Ready`. The default value of service binding state is `Malformed`.
 
 The `conditions` list of the service binding contains the following properties:
-- `Type`: The service binding condition type is `Ready`
+- `Type`: The service binding condition type is `Bound` or `NotBound`. `Bound` that the
+secret is projected into the application. `NotBound` denotes that the secret is not projected into the application. This can only occur if the secret is not found in the application namepsace.
```

Comment:
```suggestion
- `Type`: The service binding condition type is `Bound` or `NotBound`.
`Bound` means that the secret is projected into the application. 
`NotBound` denotes that the secret is not projected into the application.
This can only occur if the secret is not found in the application namespace.
```